beginner_setup_guide.md

```diff
@@ -0,0 +1,164 @@
+# Introduction
+
+The Websocket Gameboy Printer Emulator is an easy-to-build easy-to-use tool to create digital copies of your Gameboy Camera photos. It is currently still under development but already reliable and usable; it's based on [ESP8266](https://github.com/esp8266/arduino) board design. This guide is aimed at beginners and will have some redundant information. 
```

Comment:
Can we remove "websocket" everywhere?
Actually, I'm planning on renamig this repo, as there are no more websockets involved :-)

Reply:
Sure thing!